estela-web/src/components/ProxySettingsPage/index.tsx

```diff
@@ -0,0 +1,295 @@
+        setEnvVars([...envVars, {
+            name: "ESTELA_PROXY_NAME",
+            value: proxySelected ? proxySelected.name : "",
+            masked: false,
+        }])
+        setOpenBMCModal(false);
+    };
+
+    return (
+        <div>
+            {checkIfProxyExist() ? (
+                <Row className="bg-white rounded-lg my-4">
+                    <Space direction="vertical" className="lg:m-8 md:mx-6 m-4 w-full">
+                        <div className="flex flex-col">
+                            <p className="text-2xl text-black">Proxy Settings</p>
+                            <p className="text-estela-black-medium text-sm">
+                                Control and configure your proxies effortlessly.
+                            </p>
+                            <ProxyTag key="123" id={123}>
+                                {getProxyValue("ESTELA_PROXY_NAME")}
+                            </ProxyTag>
+                        </div>
+                    </Space>
+                </Row>
+            ) : (
+                <Row className="bg-white rounded-lg my-4">
+                    <Space direction="vertical" className="lg:m-8 md:mx-6 m-4 w-full">
+                        <div className="flex flex-col">
+                            <p className="text-2xl text-black">Proxy Settings</p>
+                            <p className="text-estela-black-medium text-sm">
+                                Control and configure your proxies effortlessly.
+                            </p>
+                            <div className="mt-4">
+                                <div className="">
+                                    <Row className="flex flex-col space-y-4">
+                                        <Button
+                                            onClick={() => {
+                                                setOpenProxyUserModal(true);
+                                            }}
+                                            size="large"
+                                            className="text-estela-blue-full w-96 h-24 border-0 bg-estela-blue-low text-base rounded estela-border-proxy"
+                                        >
+                                            <span className="text-center font-semibold">Manual configuration</span>
+                                            <p className="text-xs text-estela-black-full">Configure your own proxy</p>
+                                        </Button>
+                                        <Button
+                                            onClick={() => handleBMCProxies()}
+                                            size="large"
+                                            className="text-estela-blue-full w-96 h-24 border-0 bg-estela-blue-low text-base rounded estela-border-proxy"
+                                        >
+                                            <span className="text-center font-semibold">BMC Proxy</span>
```

Comment:
This should not be part of estela's code. We should inject this externally via our microfrontend.

Reply:
I updated it to be set by the user and in case of no proxies then no use.